.github/workflows/docs.yml

```diff
@@ -81,12 +81,12 @@ jobs:
       github.event_name != 'pull_request'
         || needs.docs-detect-changes.outputs.path-filter == 'true'
     env:
-      SOLIDITY_DOCS_DIR: ${{ github.workspace }}/docs/solidity
+      SOLIDITY_DOCS_DIR: ${{ github.workspace }}/docs/solidity-v1
     steps:
       - uses: actions/checkout@v2
 
```

Comment:
Let's update filters on top of the file:
```
  push:
    branches:
      - main
    paths:
      - "docs/**"
      - "solidity-v1/**"
      - "!solidity-v1/dashboard/**"
```

Reply:
bf8f2de